OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs

```diff
@@ -28,8 +28,11 @@ public static bool IsCellBuildable(this World world, CPos cell, BuildingInfo bi,
 				return false;
 
 			var tile = world.Map.MapTiles.Value[cell];
-			var rampType = world.TileSet.GetTileInfo(tile).RampType;
+			var tileInfo = world.TileSet.GetTileInfo(tile);
+			if (tileInfo == null)
+				return false;
```

Comment:
I suspect this will probably break structure placement in RA, which doesn't define metadata for a bunch of its clear tiles.